tool/common/touchid/touchid.go

```diff
@@ -82,17 +100,30 @@ func (c *touchIDDiagCommand) run(cf *CLIConf) error {
 	return nil
 }
 
-type touchIDLsCommand struct {
-	*kingpin.CmdClause
+// LsCommand implements the "touchid ls" command.
+type LsCommand struct {
+	cmd *kingpin.CmdClause
 }
 
-func newTouchIDLsCommand(app *kingpin.CmdClause) *touchIDLsCommand {
-	return &touchIDLsCommand{
-		CmdClause: app.Command("ls", "Get a list of system Touch ID credentials").Hidden(),
+// MatchesCommand returns true if LsCommand matches the given fullCommand, as
+// per [kingpin.CmdClause.FullCommand].
+// Safe even if LsCommand is nil.
+func (c *LsCommand) MatchesCommand(fullCommand string) bool {
```

Comment:
The Ls and Rm commands are not bound if touchid is not available, so I've removed the embed and changed the public interfaces to make them harder to panic. This is more important now as the commands are public and reused in two distinct binaries.

Reply:
(I could revisit the conditionally-present decision, but I decided against that for now.)